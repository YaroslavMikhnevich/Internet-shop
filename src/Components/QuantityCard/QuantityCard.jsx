import React from 'react';
import ReactDOM from 'react-dom/client';
import './QuantityCard.scss'

export default function QuantityCard(props) {
        return (
            <>
                <p className='quantity-card'>{props.quantity}</p>
            </>
        )
    }