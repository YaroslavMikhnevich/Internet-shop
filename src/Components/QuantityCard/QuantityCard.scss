.quantity-card{
    position: absolute;
    right: 7px;
    bottom: 0px;
    background-color: red;
    border-radius: 50%;
    width: 15px;
    height: 15px;
    padding: 3px;
    color: white;
}