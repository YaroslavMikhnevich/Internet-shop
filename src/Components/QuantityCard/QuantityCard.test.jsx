import React from 'react';
import renderer from 'react-test-renderer';
import QuantityCard from './QuantityCard';

describe('QuantityCard', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<QuantityCard quantity={3} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});