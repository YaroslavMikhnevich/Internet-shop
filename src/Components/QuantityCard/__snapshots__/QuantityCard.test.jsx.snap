// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuantityCard should render correctly 1`] = `
<p
  className="quantity-card"
>
  3
</p>
`;
